wavs.toml

```diff
@@ -96,6 +104,11 @@ gas_denom = "untrn"
 # ----------------------------
 [wavs]
 
+# The directory to store the data. Default is "/var/wavs"
+data = "~/wavs"
+
+active_trigger_chains = ["local", "holesky"]
```

Comment:
pain point here too for watching trigger chains (when the service is registered for it, it just ignores)